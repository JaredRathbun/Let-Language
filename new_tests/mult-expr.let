let a, b, c := 2, 3, 4 in a + b + c