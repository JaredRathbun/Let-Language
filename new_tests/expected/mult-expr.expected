Parse successful!
Result: 9